docs/UsersGuide.asciidoc

```diff
@@ -554,6 +554,19 @@ Steps 4. to 6. can be repeated for further +assert_screen+s/needles without rest
 test.
 
 
+=== Job group editor gh#2111 ===
+
+Scenarios are defined as part of a job group. The `Edit job group` button exposes the editor.
+
+By default the scenarios are listed in a table per medium, with priority and machine(s).
+Values are changed on the fly. Scenarios are removed by removing all of their machines.
```

Comment:
* I don't understand what you mean with "Values are changed on the fly".
* I think it is more common in our docs to not describe in passive tense but make suggestions to the user with "You", e.g. s/Scenarios are removed by/Remove scenarios by/